generator/src/main/java/com/faforever/neroxis/generator/cli/ParameterOptions.java

```diff
@@ -61,7 +60,7 @@ public void setTerrainSymmetry(Symmetry terrainSymmetry) {
     }
 
     @Option(names = "--biome", order = 8, description = "Texture biome for the generated map")
-    public void setBiome(String biome) {
-        this.biome = Biomes.loadBiome(biome);
+    public void setBiomeName(String biome) {
+        this.biomeName = BiomeName.getByValue(biome);
```

Comment:
This should be the actual enum as the parameter. Then you can use the completion candidates like for symmetry above